tests/conftest.py

```diff
@@ -66,9 +67,25 @@ def bundle_data_old():
     return data
 
 
+@pytest.fixture(scope='function')
+def bundle_file_1():
```

Comment:
I suggest we name the files according to usability/function or content, what do you say?

Reply:
I might be confused by the name of the file, it seems to indicate that the file is supposed to be used for something